mantle/cmd/kola/testiso.go

```diff
@@ -101,6 +100,8 @@ var (
 		"pxe-offline-install.4k.uefi",
 		"pxe-online-install.bios",
 		"pxe-online-install.4k.uefi",
+		"pxe-online-install.rootfs-appended.bios",
+		"pxe-offline-install.rootfs-appended.4k.uefi",
```

Comment:
Hmm, I don't think it's worth having two variants of this test. Just one per arch seems enough. This test adds coverage for code in the initrd, but the offline vs online distinction is purely a real root thing and shouldn't matter.